src/app/app.component.ts

```diff
@@ -7,4 +8,15 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'osk-front-demo';
+  showMenu: boolean = false;
```

Comment:
z tym menu możemy się bawić, ale ja bym proponował tym się zająć, jak będziemy robić jakieś security, ustalimy jakie mamy role userów i kto z nich ma jakie menu, to wtedy sobie wstawimy to menu w dobrym miejscu, na razie to jak dla mnie może być jakie jest, żeby się łatwiej poruszać w apce i tyle